src/Compilers/CSharp/Portable/Symbols/Source/SourceEventSymbol.cs

```diff
@@ -326,6 +326,31 @@ public sealed override bool IsVirtual
             get { return (_modifiers & DeclarationModifiers.Virtual) != 0; }
         }
 
+        internal sealed override bool IsReplace
+        {
+            get { return (_modifiers & DeclarationModifiers.Replace) != 0; }
+        }
+
+        private SourceEventSymbol _replacedBy;
+        private SourceEventSymbol _replaced;
```

Comment:
Please declare all fields at the top of the type declaration, before all the constructors.